src/DynamicRupture/Output/OutputManager.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef SEISSOL_DR_OUTPUT_MANAGER_HPP
+#define SEISSOL_DR_OUTPUT_MANAGER_HPP
+
+#include "DynamicRupture/Output/Builders/ElementWiseBuilder.hpp"
+#include "DynamicRupture/Output/Builders/PickPointBuilder.hpp"
+#include "DynamicRupture/Output/ReceiverBasedOutput.hpp"
+#include <iostream>
+#include <memory>
+
+namespace seissol::dr::output {
+class OutputManager {
+  public:
+  ~OutputManager();
+  OutputManager() = delete;
+  OutputManager(ReceiverBasedOutput* concreteImpl) : impl(concreteImpl){};
+  void setInputParam(const YAML::Node& inputData, MeshReader& userMesher);
+  void setLtsData(seissol::initializers::LTSTree* userWpTree,
+                  seissol::initializers::LTS* userWpDescr,
+                  seissol::initializers::Lut* userWpLut,
+                  seissol::initializers::LTSTree* userDrTree,
+                  seissol::initializers::DynamicRupture* userDrDescr);
+
+  void init();
+  void initFaceToLtsMap();
+  void writePickpointOutput(double time, double dt);
+  void flushPickpointDataToFile();
+  void updateElementwiseOutput();
+
+  protected:
+  bool isAtPickpoint(double time, double dt);
+  void initElementwiseOutput();
+  void initPickpointOutput();
+
+  std::unique_ptr<ElementWiseBuilder> ewOutputBuilder{nullptr};
+  std::unique_ptr<PickPointBuilder> ppOutputBuilder{nullptr};
```

Comment:
I would use the full name then:
elementWise and PickPoint.
But I guess, this is a problem because of the character limit of the automatic formatting leading to split lines.